src/app/api/googleDrive/route.ts

```diff
@@ -0,0 +1,43 @@
+import { google } from "googleapis"
+import * as googleDrive from "./googleDrive.api"
+
+const auth = new google.auth.GoogleAuth({
+  // your credentials to authenticate
+  keyFile: process.cwd() + "/src/app/config/credentials.json",
+  // the actions you are permissed to perform using this API, in this case
+  // all CRUD operations are permissed, check out
+  // [ https://developers.google.com/drive/api/guides/api-specific-auth ]
+  // for more advice on scopes
+  scopes: ["https://www.googleapis.com/auth/drive"],
+})
```

Comment:
This shall go in the service layer

Reply:
if I put in service layer the Env will be public